cfgov/filing_instruction_guide/models.py

```diff
@@ -184,4 +184,4 @@ def get_context(self, request, *args, **kwargs):
         return context
 
     class Media:
-        js = ["reports-sidenav.js"]
+        js = ["fig-sidenav.js"]
```

Comment:
Is adding this JS hooked into Wagtail or Django? Typically if a script appears in `unprocessed/apps` and then is referenced by a template (i.e. `cfgov/filing_instruction_guide/jinja2/filing_instruction_guide/index.html` in this PR), then it would not need to appear in the model Media class.